learn.json

```diff
@@ -1150,6 +1150,24 @@
 			}]
 		}]
 	},
+
+	"lit": {
+		"name": "Lit",
+		"description": "Lit is a simple library for building fast, lightweight web components.",
+		"homepage": "lit.dev",
+		"examples": [{
+			"name": "Example",
+			"url": "examples/lit/dist"
+		}],
+		"link_groups": [{
+			"heading": "Official Resources",
+			"links": [{
+				"name": "Getting Started",
+				"url": "https://lit.dev/docs/getting-started/"
+			}]
```

Comment:
Would be good to link to tutorials: https://lit.dev/tutorials/

Reply:
added!
Happy to add as many resources as you'd like - just created a boilerplate to work with